src/styles.scss

```diff
@@ -107,7 +107,10 @@ mat-card.dashboard-widget {
   padding: 10px;
 }
 
-
+.align-dashboard-table-cell{
+  padding-right: 10px;
+  text-align: right;
+}
```

Comment:
styles should usually be added to the .scss file of the component for which they are needed. That way they are easy to find during development and encapsulated, i.e. they do not accidentally change style of completely different components because the component stylesheet file only affects its own component.

Please move this to `progress-dashboard.component.scss`